### PR TITLE
refactor: Break out `usd-oracle` directory in maple-core to be a submodule dapptools project (SC-3068)

### PR DESCRIPTION
**Pull request by satyamakgec**

### CODE REVIEW

File: contracts/test/UsdOracle.t.sol [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.6.11;
+pragma experimental ABIEncoderV2;
```
Comment (deluca-mike) [inline contracts/test/UsdOracle.t.sol]: Not critical, but this can be removed now.

File: config/ci.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "language": "Solidity",
+  "sources": {
+    "contracts/interfaces/IUsdOracle.sol": {
+      "urls": ["contracts/interfaces/IUsdOracle.sol"]
+    },
```
Comment (deluca-mike) [inline config/ci.json]: This is not necessary as it's not a unique entry point with respect to ci.

File: config/dev.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "language": "Solidity",
+  "sources": {
+    "contracts/interfaces/IUsdOracle.sol": {
+      "urls": ["contracts/interfaces/IUsdOracle.sol"]
+    },
```
Comment (deluca-mike) [inline config/dev.json]: This is not necessary as it's not a unique entry point with respect to dev.

File: config/prod.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{
+  "language": "Solidity",
+  "sources": {
+    "contracts/interfaces/IUsdOracle.sol": {
+      "urls": ["contracts/interfaces/IUsdOracle.sol"]
+    },
```
Comment (deluca-mike) [inline config/prod.json]: This is not necessary as it's not a unique entry point with respect to prod.

**Review by deluca-mike — DISMISSED** [review]
The circle ci `config.yml` should still be the `dapp_test` with the 2 runs steps.

**Review by lucas-manuel — APPROVED** [review]

**Pull request merged**